terraform/application/statuscake.tf

```diff
@@ -7,4 +7,7 @@ module "statuscake" {
   ssl_urls    = compact([var.external_url])
 
   contact_groups = var.statuscake_contact_groups
+
+  heartbeat_names  = ["heartbeat-check"]
+  heartbeat_period = 600  # The period in seconds within which a heartbeat must be received
 }
```

Comment:
The devs will need an environment variable in the worker with the check URL. Please populate it in https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/master/terraform/application/application.tf#L20 using the module output